themes/ctl-histologylab/static/js/src/tiles.js

```diff
@@ -29,7 +29,7 @@ $(document).ready(function() {
 
 		var text = L.DomUtil.create('div');
 		text.id = "info_text";
-		text.innerHTML = "Magnification: Fit"
+		text.innerHTML = "Magnification: Zoom Level " + map.getZoom();
```

Comment:
Should this be 2x to match the other labels? Or, am I misunderstanding what's happening?

Reply:
@sdreher Zoom level two will have a different magnification depending on the max zoom value so I can't really hardcode 2x here